time stamp

# latest recognized object that is lying on the conveyor belt | only published when object colour changes 
uint32    object_colour # 1=red | 2=green | 3=blue | 4= reference point=> no object detected  

